rs/embedders/tests/system_api.rs

```diff
@@ -2297,11 +2297,13 @@ fn test_env_var_name_operations() {
     let mut heap = vec![0u8; 16];
 
     // Copy first variable name
-    api.ic0_env_var_name_copy(0, 0, 0, 10, &mut heap).unwrap();
+    api.ic0_env_var_name_copy(0, 0, 0, var_name_1.len(), &mut heap)
+        .unwrap();
     assert_eq!(&heap[0..10], var_name_1.as_bytes());
```

Comment:
```suggestion
    assert_eq!(&heap[0..var_name_1.len()], var_name_1.as_bytes());
```
also below, please

Reply:
The goal would be to have no integer literals in the test: they are hard to make sense of later on if the test needs to be debugged/refactored.